src/makeRobot.js

```diff
@@ -38,7 +39,64 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'wheels': wheels,
+    'version': version,
```

Comment:
you don't need quotes here and use simplified syntax for keys with identical names
```suggestion
    name,
    wheels,
    version,
```